package.json

```diff
@@ -14,21 +14,10 @@
     "url": "git+https://github.com/1-Platform/one-platform.git"
   },
   "version": "0.0.0",
-  "author": {
-    "name": "Sayak Sarkar",
-    "email": "sayak.bugsmith@gmail.com"
-  },
+  "author": "Sayak Sarkar <sayak.bugsmith@gmail.com>",
```

Comment:
@supriyagawade could you please explain how this change is related to cypress implementation?